src/main/java/application/service/OrderItemService.java

```diff
@@ -0,0 +1,15 @@
+package application.service;
+
+import application.dto.item.OrderItemResponseDto;
+import application.model.OrderItem;
+import java.util.List;
+import org.springframework.security.core.Authentication;
+
+public interface OrderItemService {
+    OrderItem save(OrderItem orderItem);

```

Comment:
Be consistent in your services regarding returning of entities or DTOs. If you use DTOs, use them for all services, but don't mix entity/DTO in a service.

Reply:
Still valid point